### ISSUE TITLE
Is it possible to filter specific execve based on the filename that is being executed?

### ISSUE THREAD

**Issue by gabrielemarra** [google/kafel#37]
I would like to ALLOW only one specific filename to be executed by an `execve` and KILL all the others.
I tried to look inside the examples or documentation but it doesn't exist anything similar. I made a few attempts but they were unsuccessful.

The specific use case of this policy would be to use it with **nsjail** to kill all the `execve` called inside the sandbox. The issue is that nsjail executes an `execve` immediately after applying the seccomp-bpf filter so it crashes.

There is already an [issue](https://github.com/google/nsjail/issues/131) for this on the nsjail repo, but it hasn't been addressed since 2020.

**Comment by robertswiecki** [comment]
Please see https://github.com/google/nsjail/issues/131#issuecomment-1881292231

TL;DR it's impossible to do that w/o using a parent process (ptrace), which we don't use with nsjail.
